website/cue/reference/cli.cue

```diff
@@ -644,6 +648,12 @@ cli: {
 				"""
 			type: bool: default: false
 		}
+		VECTOR_ALLOW_EMPTY_CONFIG: {
+			description: """
+				Allow the configuration to run without any components. This is useful for loading in an empty stub config that will later be replaced with actual components. Note that this is likely not useful without also watching for config file changes as described for `--watch-empty`.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏


```suggestion
				Allow the configuration to run without any components. This is useful for loading in an empty stub config that will later be replaced with actual components. Note that this is likely not useful without also watching for config file changes as described in `--watch-empty`.
```

Reply:
I don't think `--watch-empty` is a config option 🤔

Reply:
Fixed in 7ac746a22

Reply:
Ugh, fixed again in 77c1c2e02